pyspark/test/bigdl/keras1/test_layer.py

```diff
@@ -267,6 +268,120 @@ def test_merge_concat(self):
+        layer = ParametricSoftplus(alpha_init=0.4, beta_init=2.5, input_shape=(2, 3))
+        self.__modelTestSingleLayer(input_data, layer, functional_api=True)
+        self.__modelTestSingleLayer(input_data, layer, functional_api=False)
+
+    def test_thresholdedrelu(self):
+        input_data = np.random.random_sample([1, 2, 3])
+        layer = ThresholdedReLU(theta=0.2, input_shape=(2, 3))
+        self.__modelTestSingleLayer(input_data, layer, functional_api=True)
+        self.__modelTestSingleLayer(input_data, layer, functional_api=False)
+
+    def test_zeropadding1d(self):
+        input_data = np.random.uniform(0, 1, [3, 2, 3])
+        layer1 = ZeroPadding1D(padding=3, input_shape=(2, 3))
+        self.__modelTestSingleLayer(input_data, layer1, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer1, functional_api=True)
+        layer2 = ZeroPadding1D(padding=(2, 3), input_shape=(2, 3))
+        self.__modelTestSingleLayer(input_data, layer2, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer2, functional_api=True)
+        layer3 = ZeroPadding1D(padding={'left_pad': 1, 'right_pad': 2}, input_shape=(2, 3))
+        self.__modelTestSingleLayer(input_data, layer3, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer3, functional_api=True)
+
+    def test_zeropadding2d(self):
+        input_data = np.random.uniform(0, 1, [1, 2, 3, 4])
+        layer1 = ZeroPadding2D(padding=(2, 3), input_shape=(2, 3, 4))
+        self.__modelTestSingleLayer(input_data, layer1, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer1, functional_api=True)
+        layer2 = ZeroPadding2D(padding=(2, 3, 4, 1), input_shape=(2, 3, 4))
+        self.__modelTestSingleLayer(input_data, layer2, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer2, functional_api=True)
+        layer3 = ZeroPadding2D(padding={'top_pad': 1, 'bottom_pad': 2, 'left_pad': 3, 'right_pad': 4}, input_shape=(2, 3, 4))
+        self.__modelTestSingleLayer(input_data, layer3, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer3, functional_api=True)
+
+    def test_zeropadding3d(self):
+        input_data = np.random.uniform(0, 1, [3, 2, 4, 1, 5])
+        layer = ZeroPadding3D(padding=(1, 2, 3), input_shape=(2, 4, 1, 5))
+        self.__modelTestSingleLayer(input_data, layer, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer, functional_api=True)
+
+    def test_cropping1d(self):
+        input_data = np.random.uniform(0, 1, [3, 4, 3])
+        layer = Cropping1D(cropping=(1, 2), input_shape=(4, 3))
+        self.__modelTestSingleLayer(input_data, layer, functional_api=False)
+        self.__modelTestSingleLayer(input_data, layer, functional_api=True)
+
+    def test_simplernn(self):
+        input_data = np.random.random([3, 4, 5])
+        layer = SimpleRNN(5, input_shape=(4, 5), return_sequences=True)
```

Comment:
add more test for return_sequences=False

Reply:
add unit test for setting diff activation

Reply:
DONE